lua/swenv/api.lua

```diff
@@ -106,6 +106,15 @@ local get_conda_base_path = function()
   end
 end
 
+local get_micromamba_base_path = function()
+  local micromamba_env_path = Path:new(vim.fn.getenv('MAMBA_ROOT_PREFIX')) .. '/envs'
+  if micromamba_env_path == vim.NIL then
+    return nil
+  else
+    return micromamba_env_path
+  end
+end
```

Comment:
Shouldn't this be?
```suggestion
  local micromamba_root_prefix = vim.fn.getenv('MAMBA_ROOT_PREFIX')
  if micromamba_root_prefix == vim.NIL then
    return nil
  else
    return Path::new(micromamba_root_prefix) .. '/envs'
  end
end
```
otherwise the variable is never nil?

Reply:
Oops, sorry for the mistake, you're right! Tested and working :)